dist/index.js

```diff
@@ -9772,8 +9774,13 @@ const run = async () => {
     let versionsToDelete = await getVersionsToDelete();
     let deletedVersions = [];
     for (const version of versionsToDelete) {
-        core.debug(`will try to delete ${JSON.stringify(version)}`);
-        await deleteVersion(version);
+        if (process.env.CI === 'true') {
```

Comment:
oh nice, we can test locally

Reply:
That's the whole reason I put it in 😄 

> 
> 
> thank you very much for helping out here! I had a deeper look into githubs gql docs (https://developer.github.com/v4/object/packageversion/) and this seems to be the current way to do things. Pretty bad from them to do such a breaking change on a important feature.

Well, I agree.. partly

This was a preview API (deprecation was announced for April already). But if you're not on top of the API (and neither am I) these things are easy to miss.